content/blog/monthly-may-2020.md

```diff
@@ -0,0 +1,92 @@
+---
+title: This Month in TiKV - May 2020
+date: 2020-05-28
+author: TiKV Authors
+---
+
+As the summer tiptoed in, we were cheerful about everything it brings to us: sunshine, ice cream, and a busy but fulfilling May!
+
+Let’s take a look at what we have accomplished.
+
+## News
+
+The first CNCF ambassador spotlight goes to Queeny Jin, who has been spreading the word about TiKV to English speakers since its inception in 2016. We appreciate Queeny’s contribution. "Together we can go further!" The full blog is available [here](https://www.cncf.io/blog/2020/05/01/cncf-ambassador-spotlight-queeny-jin-of-tikv/).
```

Comment:
```suggestion
The first CNCF ambassador spotlight goes to Queeny Jin, who has been spreading the word about TiKV to English speakers since its inception in 2016. We appreciate Queeny’s contribution, and "together we can go further!" The full blog is available [here](https://www.cncf.io/blog/2020/05/01/cncf-ambassador-spotlight-queeny-jin-of-tikv/).
```